packages/mobile-app/data/index.ts

```diff
@@ -0,0 +1,7 @@
+import { createFacadeContext } from "data-facade";
+import { accountsHandlers } from "./accounts/handlers";
+
+const facadeContext = createFacadeContext(accountsHandlers);
```

Comment:
This is importing the mock implementation, how will this look when we have multiple handlers for different systems? Will there be a conditional here based on a setting? I am imagining that a user might change data backend at any point. Will the backend change result in a account reset/chain reset? Will that have any impact on your representation here?